tee-worker/ts-tests/common/utils/identity-helper.ts

```diff
@@ -61,8 +60,9 @@ export async function buildIdentityTxs(
         const identity = identities[k];
         let tx: SubmittableExtrinsic<ApiTypes>;
         let nonce: number;
-        const encod_identity = api.createType('LitentryIdentity', identity).toU8a();
-        const ciphertext_identity = encryptWithTeeShieldingKey(teeShieldingKey, encod_identity).toString('hex');
+        const encod_identity = context.sidechainRegistry.createType('LitentryPrimitivesIdentity', identity);
```

Comment:
```suggestion
        const encoded_identity = context.sidechainRegistry.createType('LitentryPrimitivesIdentity', identity);
```

Also, what's our convention for variable identifiers? `snake_case` or `camelCase`?

Reply:
I would agree - either `encoded_identity` or `enc_identity` if you want a shorter name, but not `encod` :P

Reply:
> Also, what's our convention for variable identifiers? snake_case or camelCase?

I prefer snake_case as the uniform format for variables.